### ISSUE TITLE
Country is not working properly

### ISSUE THREAD

**Issue by MHassanSaqib** [CdTgr/vue3-q-tel-input#15]
Every time when refreshing the page, the country becomes +1 regardless of the mobile number (v-model:tel="mobile") that also contains the country code i.e. +92.

**Comment by CdTgr** [comment]
@MHassanSaqib Can you share the way you were using the component?

**Comment by MHassanSaqib** [comment]
```
 <vue3-q-tel-input v-model:tel="mobile" dense outlined />
  setup() {
    const mobile = ref("+923211111111");  // number is hardcoded there. 
    return { 
          mobile,
    }
    }
```

> @MHassanSaqib Can you share the way you were using the component?

**Comment by CdTgr** [comment]
@MHassanSaqib Sorry for the delay. The issue on could be resolved in the latest release 1.0.47.
